Makefile

```diff
@@ -168,14 +168,29 @@ prune: ## Prune cached artifacts.
 csv-ceph: export MAX_DESC_LEN=0 # sets the description length to 0 since CSV cannot be bigger than 1MB
 csv-ceph: export NO_OB_OBC_VOL_GEN=true
 csv-ceph: csv-clean crds ## Generate a CSV file for OLM.
+	$(MAKE) -C images/ceph csv-clean
 	$(MAKE) -C images/ceph csv
 
-csv-clean: ## Remove existing OLM files.
-	@$(MAKE) -C images/ceph csv-clean
```

Comment:
csv-clean in general was causing changes to tracked CSVs on build, so I just remove it.